tools/buildtools/ambertools/antechamber.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="antechamber" name="AnteChamber" version="@VERSION@">
+  <description>Amber's molecular input file processor</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"></expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+    antechamber -i '$input1' -o '$output1'
+    #if $input1.is_of_type("pdb")
+    -fi pdb
+    #else if $input1.is_of_type("mol2")
+    -fi mol2
```

Comment:
You could replace this conditional with `-fi $input1.ext` which is slightly more compact, but I don't think it's essential.

Reply:
now that ftype in place, yes!